version.properties

```diff
@@ -1,12 +1,12 @@
-#Fri, 20 Aug 2021 15:18:38 +0200
+#Mon, 23 Aug 2021 10:29:29 +0200
 
 # WordPress version information
-wordpress.versionName=18.0
-wordpress.versionCode=1098
+wordpress.versionName=18.1-rc-1
+wordpress.versionCode=1099
 
-wordpress.zalpha.versionName=alpha-314
-wordpress.zalpha.versionCode=1097
+wordpress.zalpha.versionName=alpha-315
+wordpress.zalpha.versionCode=1100
 
 # Jetpack version information
-jetpack.versionName=18.0
-jetpack.versionCode=1096
+jetpack.versionName=18.1-rc-1
```

Comment:
I'm not familiar with the Jetpack setup, so: why the old version code is different from the one for WordPress (`1096` vs `1098`)?

Reply:
Because paaHJt-2s8-p2 is still not finished 😉 

That's the whole issue with how we've been managing those versions automatically so far after Javon's changes in the 1.3.0 toolkit: versionName / versionCode for WP and JP are still handled and bumped separately by the toolkit.  
And since WP has both an alpha and a beta, bumping versionCodes 2-by-2, while JP only has beta and bumps its versionCodes 1-by-1, that creates that drift.

I sometimes realign the versionCodes manually from time to time, but it's still not great to not have that automated and them using the same code for both… hence why I'm eager to get this paaHJt-2s8-p2 project done. (Draft for fixing this [here](https://github.com/wordpress-mobile/WordPress-Android/pull/15230))

Reply:
Just to give you an idea of how it went with last releases because of the current state of automation that project plans to fix:

| versionName | WP prod/beta | WP alpha | JP prod/beta | Comment |
|:--|--:|--:|--:|:--|
| 17.9 | 1090 | - | 1086 | Existing drift from previous betas |
| 18.0-rc-1 | 1091 | 1092 | 1091 | JP/WP versionCodes manually realigned on code freeze |
| 17.9.1 | 1093 | - | 1093 | Manually fixed JP versionCode for the hotfix to be 1093 like WP and not just 1091+1 |
| 18.0-rc-2 | 1094 | 1095 | 1094 |
| 18.0-rc-3 | 1096 | 1097 | 1095 | WP drift as it has no alpha and is bumped independently from WP |
| 18.0 | 1098 | - | 1096 | |
| 18.1-rc-1 | 1099 | 1100 | 1099 | Realigned manually on code freeze |